assets/oag.css

```diff
@@ -198,101 +355,310 @@
+		margin-bottom: 20px;
+		overflow: hidden;
+        padding: 20px;
+		position: relative;
+		transition: all ease .5s;
+
+		& > img:first-child {
+			display: block;
+			margin: -20px -20px 20px -20px; 
+			width: calc(100% + 40px); /* 2 × padding */
+		}
+
+		.label + time {
+			margin-left: 1em;
+		}
+
+		time {
+			font-weight: 700;
+			font-size: 75%;
+		}
+
+		h2, h3, h4 {
+			margin-top: 0;
+
+			a {
+				text-decoration: none;
+			}
+		}
+
+		&.bg-light {
+			box-shadow: var(--oag-shadow);
+		}
+
+		&.oag-heading-as-body {
+			h2, h3, h4 {
+				font-family: var(--oag-body-font);
+				font-size: 1em;
+				font-weight: normal;
+
+				a {
+					color: var(--oag-body-color);
+					text-decoration: none;
+
+					&:hover {
+						color: var(--oag-primary-color);
+					}
+				}
+			}
+		}
+
+		&.bg-light:has(.stretched-link):hover {
+			box-shadow: var(--oag-shadow-lg);
+		}
+	}
+
+	.wb-paginate-pager {
+		.pagination {
+			> li {
+				> button {
+					color: var(--oag-primary-color);
+				}
+
+				&.active {
+					> button {
+						background-color: var(--oag-primary-color);
+						border-color: var(--oag-primary-color);;
+						color: #fff;
+					}
+				}
+			}
+		}
+	}
+
+
+	/**********
+		Utilities
+	**********/
+
 	.text-primary {
 		color: var(--oag-primary-color);
-	}		
-	
-	.lead {
-		font-size: 2em;
-		font-weight: 100;
-		line-height: 1.2;
 	}
 
-	.well {
-		border: 2px solid #908F84;
-	}	
+	.text-white {
+		color: #fff!important;
+	}
+
+	.fnt-hdng {
+		font-family: var(--oag-heading-font);
+	}
 
-	.gc-srvinfo a {
-		color: var(--oag-primary-color);
+	.bg-light {
+		background-color: var(--oag-light-bg-color) !important;
+	}
+
+	.oag-rds {
+		border-radius:var(--oag-rds);
+	}
+
+	.oag-rds-sm {
+		border-radius:var(--oag-rds-sm);
+	}
+
+	.oag-brdr {
+		border: 1px solid var(--oag-default-border-color);
+	}
+
+	.oag-shadow {
+		box-shadow: var(--oag-shadow);
 	}
-	.panel-default {
-		/* Default panel styling */
-		border: 1px solid transparent;
-		background-color: #F4F4F2;
+
+	/**********
+	Home Page
+	**********/
+	&.home {
+		#wb-bnr + hr {
+			border-color: #ddd;
+		}
+
+		.gcweb-menu button[aria-haspopup=true] {
+			background-color: #fff;
+			border-color: #fff;
+			color: var(--oag-primary-color);
+
+			&:hover, &[aria-expanded="true"] {
+				background-color: var(--oag-primary-color);
+				color: #fff;
+			}
+		}
+
+		h2 {
+			font-size: 2.4375rem;
+		}
+
+		.lead {
+			color: #fff;
+			font-family: var(--oag-heading-font);
+			font-size: 3em;
+			padding: 1em 0 2em;
+		}
+
+		.reports {
+			margin-top: -4em;
+		}
+
+		.oag-card {
+			&.oag-brdr {		 
+				border: 2px solid var(--oag-default-border-color);		
+			}
+
+			.alert {
+				background: none;
+
+				time {
+					color: var(--oag-body-color);
+					font-family: var(--oag-heading-font);
+					font-size: 2.5625rem;
+					margin: 0;
+				}
+			}
+		}
 	}
 	
+	/**********
+		Extra-small screens
+	**********/
 	@media (max-width: 767px) {
 		.lead {
 			font-size: 1em;
 		}		
```

Comment:
Is this style working? I think it get overwritten by the other media query because it is defined after. Am I right?